### PR TITLE
Revamp dodge

### PR DESCRIPTION
**Pull request by MaxWofford**

### CODE REVIEW

File: workshops/dodge/README.md [hunk 1]
```diff
@@ -1,13 +1,25 @@
...
+![Final result](img/after_spicing_it_up.gif)
+
+Here's the [live demo][final_live_demo].
+
+Also, here's the [final code][final_code].
+
+This workshop should take around 1 hour and 15 minutes.
+
+[final_live_demo]: https://prophetorpheus.github.io/dodge
+Workshop revisions: On track to finish up the core workshops today
```
Comment (MaxWofford) [inline workshops/dodge/README.md]: Fuck

Comment (MaxWofford) [inline workshops/dodge/README.md]: oh_no

File: workshops/dodge/README.md [hunk 2]
```diff
@@ -1,13 +1,25 @@
...
+Running late on lecture-hall bugs, but just waiting for PRs to be reviewed now
```
Comment (MaxWofford) [inline workshops/dodge/README.md]: tumblr_nympbmpept1r62ipxo4_250

**Comment by MaxWofford** [comment]
115862-yes-this-is-doge-you-buy-pr-no-ok-bye

**Comment by MaxWofford** [comment]
@paked Can you review this?

**Comment by paked** [comment]
@MaxWofford Sure!

**Comment by paked** [comment]
LGTM.

**Pull request merged**